query_timetable.py

```diff
@@ -0,0 +1,34 @@
+import json
```

Comment:
Yeah that's fine!  The reason I don't usually put shebangs on python scripts is because I generally prefer calling python explicitly: that way, no matter where people put the executables, or which exact version of python they use it's fine either way.  That being said, it doesn't hurt at all, so I'm gonna commit this change.